osf/utils/datetime_aware_jsonfield.py

```diff
@@ -12,13 +12,23 @@
 from psycopg2.extras import Json
 
 
+class NaiveDatetimeException(BaseException):
```

Comment:
Also, it might make sense to move this to `osf.exceptions`. That's minor, though.